### DISCUSSION TITLE
Need instructions on adjusting tray icon

### DISCUSSION THREAD

**Discussion by myfix16** [HavenDV/H.NotifyIcon#65] [Q&A] [answered]
Hi,
I am migrating my battery utility to Windows App SDK. But the font looks small and not centered. Below are the appearance of the icon and my code to create it:

<img width="145" alt="image" src="https://user-images.githubusercontent.com/61649477/212106898-5e53fe8d-6fc0-490f-bd5d-0b8fad322fa5.png">

```
<tb:TaskbarIcon.GeneratedIcon>
    <tb:GeneratedIcon
        Text="80"
        TextMargin="0 0 0 0"
        Margin="0 0 0 0"
        CornerRadius="0"
        Foreground="White"
        FontWeight="SemiBold"
        FontSize="48"
        FontFamily="Segoe UI"
    />               
</tb:TaskbarIcon.GeneratedIcon>
```

I cannot set the `FontSize` above 48, which will result in a blank icon. So how can I achieve the same effect as the left icon shown in the picture? 

More generally, it would be better if more documentation on the taskbar icon visual adjustment could be added to the repo😀. 

Thank you!

**Comment by HavenDV** [comment, marked as answer]
Hi. I've already done some work on this, take a look here: https://github.com/HavenDV/H.NotifyIcon/issues/39
Try this code:
```xml
<tb:TaskbarIcon.GeneratedIcon>
    <tb:GeneratedIcon
        Text="80"
        Foreground="White"
        FontSize="55"
        FontFamily="Arial"
    />               
</tb:TaskbarIcon.GeneratedIcon>
```

**Comment by myfix16** [comment]
Cool, that works! 

However, I have several questions:
1. Other properties remain the same, why `FontFamily=Arial` works but `FontFamily=Segoe UI` results in a blank icon?
2. I find the icon a bit aliased, is there anything I can do?
<img width="190" alt="image" src="https://user-images.githubusercontent.com/61649477/212122263-ee555a10-e607-404b-8c6a-ebad4168b554.png">

**Reply by HavenDV** [reply]
You can try to increase Size to 512 and FontSize to 220, it may improve the quality.
All available properties are available here:
icons.https://github.com/HavenDV/H.NotifyIcon/blob/master/src/libs/H.NotifyIcon.Shared/GeneratedIcon.cs
Because System.Drawing is used, there are a series of restrictions on generated
